tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package main
+
+import (
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/metrics"
+	"github.com/kubevirt/monitoring/test/metrics/prom-metrics-linter/metrics-parser"
```

Comment:
Please rename to avoid underscore names bellow.

e.g.
```suggestion
	parser "github.com/kubevirt/monitoring/test/metrics/prom-metrics-linter/metrics-parser"
```

or even
```suggestion
	metricsparser "github.com/kubevirt/monitoring/test/metrics/prom-metrics-linter/metrics-parser"
```

Reply:
Done